frontend/src/components/TaskCard.tsx

```diff
@@ -0,0 +1,172 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import Card from './Card'
+import { Header, Subheader, Paragraph } from './Typography'
+import Button from './Button'
+
+const ExpandedTaskCardWrapper = styled.div`
+    padding: 0px 1.25em 1.25em 1.25em
```

Comment:
Let's move the left-right padding to the Task List component rather than individual task cards. This will properly align the header 'Tasks' inside Task List as well.